### PR TITLE
xz_utils: fix pkg-config file name

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **lib/1.0**

- [ ] I've read the [guidelines](https://github.com/conan-io/conan-center-index/wiki#how-to-submit-a-pull-request) for contributing.
- [ ] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [ ] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [ ] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by ericLemanissier** [comment]
@danimtb what do you think?

File: recipes/xz_utils/all/conanfile.py [hunk 1]
```diff
@@ -131,3 +131,4 @@ def package_info(self):
         if not self.options.shared:
             self.cpp_info.defines.append('LZMA_API_STATIC')
         self.cpp_info.libs = tools.collect_libs(self)
+        self.cpp_info.names['pkg_config'] = 'lzma'
```
Comment (danimtb) [inline recipes/xz_utils/all/conanfile.py]: is this really the right name for this recipe? Could you provide more info about this

**Comment by ericLemanissier** [comment]
After double-checking, it looks like lzma is not the pkg-config file name, but only the library name. It looks like the upstream library does not provide any .pc file anyway. sorry for the noise.

**Comment by uilianries** [comment]
@ericLemanissier The author provides, but the name is `liblzma.pc` I just checked now: https://gist.github.com/uilianries/b9bd58cbaed61209849d8b823d215dd5

**Comment by uilianries** [comment]
Let's restart

File: recipes/xz_utils/all/conanfile.py (hunk 1 above)
Comment (uilianries) [reply recipes/xz_utils/all/conanfile.py]: hey @danimtb I checked it here: https://github.com/conan-io/conan-center-index/pull/446#issuecomment-567901865

**Comment by ericLemanissier** [comment]
thanks @uilianries I don't know how I could miss this file in the sources. Windows' research tool looks broken....

**Pull request merged**